src/server/pfs/server/commit_store.go

```diff
@@ -256,24 +256,3 @@ func commitDiffTrackerID(commit *pfs.Commit, fs fileset.ID) string {
 func commitTotalTrackerID(commit *pfs.Commit, fs fileset.ID) string {
 	return commitTrackerPrefix + pfsdb.CommitKey(commit) + "/total/" + fs.HexString()
 }
-
-// SetupPostgresCommitStoreV0 runs SQL to setup the commit store.
-// DO NOT MODIFY THIS FUNCTION
-// IT HAS BEEN USED IN A RELEASED MIGRATION
-func SetupPostgresCommitStoreV0(ctx context.Context, tx *pachsql.Tx) error {
```

Comment:
Where did this go?

Reply:
I am not sure, I don't think this should be getting deleted